src/st-trace/trace.c

```diff
@@ -98,13 +98,53 @@ static void usage(void) {
   puts("  -V, --version         Print this version");
   puts("  -vXX, --verbose=XX    Specify a specific verbosity level (0..99)");
   puts("  -v, --verbose         Specify a generally verbose logging");
-  puts("  -cXX, --clock=XX      Specify the core frequency in MHz");
-  puts("  -tXX, --trace=XX      Specify the trace frequency in Hz");
+  puts("  -cXX, --clock=XX      Specify the core frequency");
```

Comment:
Another (more favourable) option would be to keep the unit Hz here and make solely k, M, G part of the input string.
Though not technically relevant here, one should keep in mind that "m" stands for "milli" and "M" for "mega" so looking at this it would give me a better feeling to follow the common convention here.

Reply:
There is not much backwards compatibility to care about. The st-trace tool is not in the codebase for very long.
Just don't bother about it - we'll add the change to the changelog and that should do. :wink:

Reply:
Thanks, but take your time - we're not in a hurry. :wink: